common/api/core-common.api.md

```diff
@@ -2906,16 +2906,6 @@ export enum ECSqlValueType {
     StructArray = 15
 }
 
```

Comment:
@aruniverse I don't understand why this file updated, or core-frontent.api.md, core-quantity.api.md, eschema-metadata.api.md, core-common.exports.csv, or core-frontend.exports.csv. As far as I know, I didn't make these changes, although I did merge changes from origin/master. Did some other PR get force merged without extract-api running?